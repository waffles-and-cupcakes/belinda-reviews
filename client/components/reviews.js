import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

class Reviews extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      total: 5,
      reviews: []
    }
  }

  componentDidMount() {
    axios.get('/rooms/1/reviews')
    .then((res) => {
      console.log('RESPONSE: ', res);
      this.setState({
        reviews: res.data
      });
    })
    .catch((err) => {
      console.log('error GET-ing /rooms/1/reviews')
    })
  }

  render() {
    return (
      <div> {JSON.stringify(this.state.reviews)}</div>
    );
  }
}

export default Reviews;