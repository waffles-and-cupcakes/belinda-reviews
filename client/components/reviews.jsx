import React from 'react';
import ReactDOM from 'react-dom';


class Reviews extends React.Component {

  render() {
    return (
      <section className="review-container"> 

        <div className="detail-container">
          <img className="user-photo" src="./pfp.jpg" />
          <div className="x-detail-container">
            <p> Kristen </p>
            <p> June 2017 </p>
          </div>
        </div>

        <div className="review-body"> 
          A beautiful space! We loved all the attention to 
          detail and local recommendations. Would love to se all
            their properties!
        </div>

      </section> 
    )
    
  }
}

export default Reviews;