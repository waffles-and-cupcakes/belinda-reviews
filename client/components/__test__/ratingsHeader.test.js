import React from 'react';
import {shallow} from 'enzyme';
import RatingsHeader from '../ratingsHeader';
import renderer from 'react-test-renderer';
import reviews from '../../mockData';


test('RatingsHeader renders', () => {
  const component = renderer.create(
    <RatingsHeader reviews={reviews} filterReviews={jest.fn()}/>
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});