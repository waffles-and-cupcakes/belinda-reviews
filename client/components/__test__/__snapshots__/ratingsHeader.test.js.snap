// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingsHeader renders 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
     
    <div
      className="divider"
    />
  </div>
  <div
    className="row"
  >
    <div
      className="col s12 m8 rev-title"
    >
      <span
        className="title-text"
      >
        26
         Reviews
      </span>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="input-field col s12 m4"
    >
      <input
        className="rev-search"
        onChange={[Function]}
        onKeyPress={[Function]}
        placeholder="Search reviews"
        value=""
      />
    </div>
  </div>
  <div
    className="row"
  >
     
    <div
      className="divider"
    />
  </div>
  <div
    className="row avgs"
  >
    <div
      className="col s12 m6 rating"
    >
      <div>
        Accuracy
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="col s12 m6 rating"
    >
      <div>
        Location
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="col s12 m6 rating"
    >
      <div>
        Communication
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="col s12 m6 rating"
    >
      <div>
        Check In
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="col s12 m6 rating"
    >
      <div>
        Cleanliness
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
    <div
      className="col s12 m6 rating"
    >
      <div>
        Value
      </div>
      <div>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
        <i
          className="material-icons small icon-star"
        >
          star_border
        </i>
      </div>
    </div>
  </div>
</div>
`;
